package.json

```diff
@@ -64,6 +64,7 @@
       "./nextjs-react/*",
       "./nextjs-ssr",
       "./nextjs-ssr/*",
+      "./nextjs-ssr-react-query",
```

Comment:
You need to add here name of your root package.json. 
So, for now you have two ways, update your sample root package.json
`/module-federation-examples/nextjs-ssr-react-query/package.json`
to

```
{
  "name": "nextjs-ssr-react-query",
  "version": "0.0.0",
  "license": "MIT",
  "scripts": {},
  "private": true,
  "dependencies": {
    "@module-federation/nextjs-mf": "^6.1.3",
    //...
```

or  add here name from package.json witch you already have, it's `acme`

But I prefer first method, it's more clear😀

Reply:
Solved :rocket: